### ISSUE TITLE
New-MgEntitlementManagementAccessPackageAssignment no longer needs to set start date 

### ISSUE THREAD

**Issue by markwahl-msft** [microsoftgraph/msgraph-sdk-powershell#1726]
In Entra Identity Governance entitlement management, it is no longer necessary when creating an assignment request to supply a start date.   If the policy has a custom start and end dates, the policy's settings will be authoritative.  Therefore this code in the custom cmdlet that supplied the current date as a start date should be removed, so that the policy's defaults apply if the caller does not specify an explicit start date.

```
diff --git a/src/Identity.Governance/Identity.Governance/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1 b/src/Identity.Governance/Identity.Governance/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1
index fd872d8c5..847a155d8 100644
--- a/src/Identity.Governance/Identity.Governance/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1
+++ b/src/Identity.Governance/Identity.Governance/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1
@@ -155,12 +155,6 @@ begin {
     $notDelivered = 0
     $nonUsers = 0

-    if ($null -eq $StartDate  -or $StartDate.Length -eq 0) {
-        $now = Get-Date
-        $ts = Get-Date $now.ToUniversalTime() -format "s"
-        $StartDate = $ts + "Z"
-    }
-
     if ($null -eq $Justification) {
         $Justification = ""
     }
```

**Comment by peombwa** [comment]
Thanks for the heads up!

We will remove the start date in our upcoming releases.

Note to us:
Only set `-StartDate` when a customer explicitly sets it via PSBoundParameters:
https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/7d0319fb08a64278cb8130e904a8b2d9f3f30df4/src/Identity.Governance/Identity.Governance/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1#L245-L247

**Comment by peombwa** [comment]
Reopening https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/1726.

The change should also be applied to the v2 branch at https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/features/2.0/src/Identity.Governance/beta/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1#L157-L161

**Comment by peombwa** [comment]
> Reopening #1726.
> 
> The change should also be applied to the v2 branch at https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/features/2.0/src/Identity.Governance/beta/custom/New-MgEntitlementManagementAccessPackageAssignment.ps1#L157-L161

Fixed by #1949
